build/build.sh

```diff
@@ -16,11 +16,15 @@
 
 set -e
 
+declare -A arches=( ["amd64"]="x86_64" ["arm"]="armv7l" ["arm64"]="aarch64" ["ppc64le"]="ppc64le" ["s390x"]="s390x" )
```

Comment:
As discussed offline, let's just skip to amd64 armv7 and arm64 since these are the most popular ones and we can try to test.

Reply:
Updated this to only include the architectures listed above.